pkgs/development/libraries/libwacom/default.nix

```
@@ -1,17 +1,20 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, glib, pkgconfig, udev, libgudev }:
+{ stdenv, fetchFromGitHub, meson, ninja, glib, pkgconfig, udev, libgudev }:
 
 stdenv.mkDerivation rec {
   pname = "libwacom";
-  version = "0.33";
+  version = "1.1";
```

```suggestion
  version = "1.1";
  
  outputs = [ "out" "dev" ];
```

makes it a split package.

---

Oh, awesome. Done.